scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -487,3 +477,27 @@ def parse_args(o3de_context: exp.O3DEScriptExportContext):
     except exp.ExportProjectError as err:
         print(err)
         sys.exit(1)
+
+# This code is only run by the 'export-project' O3DE CLI command
+
+if "o3de_context" in globals():
+    import pprint
+    pp = pprint.PrettyPrinter(indent=3)
```

Comment:
What is the pretty printer for? I don't see any uses

Reply:
stale code, I'll remove it